packages/frontend/src/components/merchants/listings/EditListing.tsx

```diff
@@ -74,55 +79,57 @@ export default function EditProduct() {
     }
   }, [baseToken]);
 
-  async function create(newItem: Partial<Listing>) {
+  async function create(newItem: Partial<TListing>) {
     try {
-      const { id } = await clientStateManager!.stateManager!.listings
-        .create(
-          newItem,
-          baseToken.decimals,
-        );
-      await clientStateManager!.stateManager!.listings.changeInventory(
-        id,
-        units,
-      );
+      await clientStateManager!.stateManager.set("Listing", newItem);
+      // await clientStateManager!.stateManager!.listings.changeInventory(
+      //   id,
+      //   units,
+      // );
+      await clientStateManager!.stateManager!.set("ChangeInventory", {
```

Comment:
i'm not sure how to fold this best into the api but on the patch/network level we support `increment` for deltas as well as `replace` for new values.